src/main/play-doc/developer/ConductrSandboxCluster.md

```diff
@@ -0,0 +1,70 @@
+# ConductR sandbox cluster
+
+In order to manage a ConductR cluster we provide a sbt plugin [sbt-conductr-sandbox](https://github.com/typesafehub/sbt-conductr-sandbox). Its purpose is to simplify the setup of a ConductR cluster during development or on a continous integration platform.  By default it uses the conductr-dev Docker image which gets automatically pulled down from Typesafe's public docker repository on bintray when starting the cluster.
+
+> The docker image conductr-dev contains a single node version of ConductR. It is not recommended to use this version in production.
+
+The following description is intended to provide a taste of what `sbt-conductr-sandbox` can do for you. Please refer to [its documentation](https://github.com/typesafehub/sbt-conductr-sandbox) for more details.
+
+## Setting up sbt-conductr-sandbox
+
+1. Add the sbt plugin to the `project/plugins.sbt` of your project:
+
+    ```scala
+    addSbtPlugin("com.typesafe.conductr" % "sbt-conductr-sandbox" % "1.0.2")
+    ```
+2. Enable the sbt plugin in the `build.sbt`:    
+
+    ```scala
+    lazy val root = (project in file(".")).enablePlugins(ConductRSandbox)
+    ```
+
+## Using sbt-conductr-sandbox
+
+### Starting ConductR Cluster
+
+Start the cluster inside the sbt session with:
+
+```scala
+[my-app] sandbox run
+[info] Running ConductR...
+[info] Running container cond-0 exposing 192.168.59.103:9000...
+```
+
+> Note that the ConductR cluster will take a few seconds to become available and so any initial command that you send to it may not work immediately.
+
+Given the above you will then have a ConductR process running in the background (there will be an initial download cost for Docker to download the conductr/conductr-dev image from the public Docker registry).
+
+#### ConductR features
+
+ConductR gets shipped with features which can be optionally enabled during startup by specifying the `--withFeatures` option, e.g.:
```

Comment:
It is important to mention how, if you're using boot2docker, then you'll need to increase its memory size to at least 2GB. Here's how, at least on the Mac:

```
boot2docker stop
VBoxManage modifyvm boot2docker-vm --memory 2048 
boot2docker start
```